integration/base/terraform/expected/.ship/release.yml

```diff
@@ -3,13 +3,15 @@ assets:
     - terraform:
         dest: ./terraform/random.tf
         inline: |
-          resource "random_id" "server" {
-            byte_length = {{repl ConfigOption "id_length" | ParseInt}}
+          resource "local_file" "foo" {
+            content     = "foo!"
+            filename = "/tmp/foo.bar"
```

Comment:
do we no longer support template functions here?

Reply:
we do, I was just trying to make these more reproducible. I can actually roll some of that back now that we're restoring the state